shell/src/chain_manager.rs

```diff
@@ -226,7 +226,7 @@ pub struct ChainManager {
     identity_peer_id: CryptoboxPublicKeyHash,
 
     /// Holds the state of all peers
-    peers: HashMap<ActorUri, PeerState>,
+    peers: HashMap<PeerAddress, PeerState>,
```

Comment:
@binier 
I am not sure about this, if SocketAddress is enought, I could still have more peers with the same identity?

Reply:
`TezedgeState` makes sure to not accept any peers with same identity:

https://github.com/binier/tezedge/blob/534a29e8c2742855e49b0da61621c98146b82011/tezedge_state/src/acceptors/peer_handshake_message_acceptor.rs#L58-L74